pylint_odoo/misc.py

```diff
@@ -44,34 +48,26 @@ def join_node_args_kwargs(node):
     return args
 
 
-# TODO: Change all methods here
-
-class WrapperModuleChecker(BaseChecker):
+class PylintOdooChecker(BaseChecker):
 
+    # Auto call to `process_tokens` method
     __implements__ = IAstroidChecker
 
-    node = None
-    module_path = None
-    msg_args = None
-    msg_code = None
-    msg_name_key = None
     odoo_node = None
     odoo_module_name = None
     manifest_file = None
-    module = None
     manifest_dict = None
-    is_main_odoo_module = None
 
-    def get_manifest_file(self, node_file):
-        """Get manifest file path
-        :param node_file: String with full path of a python module file.
-        :return: Full path of manifest file if exists else return None"""
-        if os.path.basename(node_file) == '__init__.py':
-            for manifest_basename in settings.MANIFEST_FILES:
-                manifest_file = os.path.join(
-                    os.path.dirname(node_file), manifest_basename)
-                if os.path.isfile(manifest_file):
-                    return manifest_file
+    def set_caches(self):
```

Comment:
Could you reduce the changes to minimal ones?
I mean, the method `set_caches` was changed of position from line 99 to line 61 but is not need it

Reply:
Done, I keep only the change necessary